jaxley/solver_voltage.py

```diff
@@ -62,10 +63,10 @@ def step_voltage_explicit(
         debug_states,
     )
     new_voltates = voltages + delta_t * update
-    return new_voltates
+    return new_voltates.ravel(order="C")
 
 
-def step_voltage_implicit(
+def step_voltage_implicit_with_custom_spsolve(
```

Comment:
if you want you could also replace custom with jaxley ;)

Reply:
jaxley_spsolve vs jax_spsolve

Reply:
Agreed!